### PR TITLE
fix null pointer exception for i18nAction.

### PR DESCRIPTION
**Pull request by vietnguyen**

### CODE REVIEW

File: dhis-2/dhis-web/dhis-web-commons/src/main/java/org/hisp/dhis/util/TranslationUtils.java [hunk 1]
```diff
@@ -59,13 +60,19 @@
...
+        if ( translations == null || locale == null )
+        {
+            return null;
+        }
+
         Map<String, String> translationMap = new Hashtable<>();
 
         for ( ObjectTranslation translation : translations )
         {
-            if ( translation.getProperty() != null && translation.getValue() != null && translation.getLocale().equals( locale ))
```
Comment (larshelge) [inline dhis-2/dhis-web/dhis-web-commons/src/main/java/org/hisp/dhis/util/TranslationUtils.java]: is this right?

Comment (vietnguyen) [inline dhis-2/dhis-web/dhis-web-commons/src/main/java/org/hisp/dhis/util/TranslationUtils.java]: Ok, property and locale are mandatory so no need to check null. 
Updated as below.

Comment (larshelge) [inline dhis-2/dhis-web/dhis-web-commons/src/main/java/org/hisp/dhis/util/TranslationUtils.java]: I mean comparing translation.getLocale() to locale.getLanguage() - is that correct? What if we have country?

Comment (vietnguyen) [inline dhis-2/dhis-web/dhis-web-commons/src/main/java/org/hisp/dhis/util/TranslationUtils.java]: Ok, I see, didn't pay attention to that. updated now

**Pull request merged**